ntbs-service/Pages/Shared/_Header.cshtml

```diff
@@ -98,9 +98,18 @@
                             </a>
                             <ul ref="navigation-submenu" v-if="showMenu" class="nav-submenu-list">
                                 <li class="header-navigation-link nav-submenu-list-item">
-                                    <a class="govuk-header__link header-navigation-list-item" href="/Help">
-                                        FAQ
-                                    </a>
+                                    @if (string.IsNullOrEmpty(_externalLinksService.GetSharePointFaqPageUrl()))
+                                    {
+                                        <a class="govuk-header__link header-navigation-list-item" href="/Help">
```

Comment:
This link will no longer work, as we've removed the help page. Do you think we need this case? We could just send the user to a "not found" or similar page.
Might be worth searching the codebase for any more links to help just in case too!

Reply:
I guess this might actually already result in a "not found" page, since the route doesn't exist?

But might be simpler to just remove this case entirely - we're committed to having an FAQ page in Sharepoint now, so having a backup option doesn't seem necessary.

Reply:
I have checked the codebase for any other usages of the help page, and this is the last of them